webapp/app.js

```diff
@@ -0,0 +1,12 @@
+import express from "express"
+import queryParser from "./middlewares/queryParser"
+import cookieParser from "./middlewares/cookieParser"
+
+const app = express();
+const port = 8090;
+
+app.listen(port, () => console.log(`App listening on port ${port}`));
+app.use(cookieParser);
+app.use(queryParser);
```

Comment:
All middleware should be applied before the `listen` because it may not be applied properly